src/helm/benchmark/static/benchmarking.js

```diff
@@ -241,7 +241,7 @@ $(function () {
       if (metricValue !== undefined) {
         const displayName = schema.metricsField(metricName).display_name;
         const statClass = getStatClass(metricName, metricValue);
-        list.push($('<span>', {class: statClass}).append(`${displayName}: ${round(metricValue, 3)}`));
+        list.push($('<span>', {class: statClass}).text(`${displayName}: ${round(metricValue, 3)}`));
```

Comment:
Is `text` supposed to be faster than `append`?

Reply:
Yes, because append requires a HTML parse and text does not. Also text is more semantically correct here (we want HTML in `displayName` to be escaped).